src/App.jsx

```diff
@@ -14,70 +16,65 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectProduct, setSelectProduct] = useState('Jam');
```

Comment:
please don't use verbs in state variable names
```suggestion
  const [selectedProduct, setSelectedProduct] = useState('Jam');
```